tsconfig.json

```diff
@@ -1,25 +1,8 @@
 {
+  "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "target": "ES6",
     "allowJs": true,
-    "jsx": "react-jsx",
-    "module": "commonjs",
-    "skipLibCheck": true,
     "esModuleInterop": true,
-    "noImplicitAny": true,
-    "sourceMap": true,
-    "baseUrl": ".",
-    "outDir": "dist",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "paths": {
-      "*": ["node_modules/*"],
-      "@Assets/*": ["src/static/*"],
-      "@Renderer/*": ["src/renderer/*"],
-      "@Types/*": ["src/renderer/types/*"],
-      "@appigram/react-rangeslider": ["./node_modules/@types/react-rangeslider"]
-    }
-  },
-  "include": ["src/**/*"],
-  "files": ["src/renderer/theme/styled.d.ts"]
```

Comment:
This line is removed and `include` includes it.
This is likely introduced when the file is drag or renamed on vscode.